prov/efa/src/rxr/rxr_pkt_type_misc.c

```diff
@@ -490,11 +490,23 @@ void rxr_pkt_handle_eor_recv(struct rxr_ep *ep,
 {
 	struct rxr_eor_hdr *shm_eor;
 	struct rxr_tx_entry *tx_entry;
+	int i, ret;
 
 	shm_eor = (struct rxr_eor_hdr *)pkt_entry->pkt;
```

Comment:
Isn't `rxr_pkt_handle_eor_recv`used for both the shm and non-shm paths? How is this directly treating it as a shm eor packet?

Reply:
Your are right, thank you, will fix it.